### DISCUSSION TITLE
EagerMOT

### DISCUSSION THREAD

**Discussion by 6Superman6** [community/community#36412] [Other Feature Feedback, Questions, & Ideas]
How to download detections_segmentations_RRC_BB2SegNet

**Comment by airtower-luna** [comment]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/aleksandrkim61/EagerMOT? In that case you should open an issue *in that repository* so the developers see it. I strongly recommend including more details about what you want, vague questions are rather impolite. :sweat_smile:

**Comment by ossanna16** [comment]
Hey there @6Superman6 and thank you for asking a good question! I'm glad you found our community 🙂
